src/scripts/main.js

```diff
@@ -1,3 +1,50 @@
 'use strict';
 
-// write code here
+const listOfElements = document.getElementsByTagName('li');
+const parentList = document.querySelector('ul');
+
+function sortList(list) {
+  const itemsArray = Array.from(list);
```

Comment:
Ensure that `list` is an array before sorting. If `list` is a live HTMLCollection, convert it using `Array.from(list)` to avoid issues with live updates.